Header header
string[] detection_types
geometry_msgs/Point[] detection_positions
